1-org/envs/shared/org_policy.tf

```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
   parent = "organizations/${var.org_id}"
   title  = "default policy"
 }
+
+
+module "restrict_procotol_fowarding" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = ["internal"]
+  enforce         = "true"
+  constraint      = "constraints/compute.restrictProtocolForwardingCreationForTypes"
```

Comment:
each new org policy should be placed in the correct section base in its type: `compute` .`iam` or `essentialcontacts`.
`essentialcontacts` can be grouped with the domain restrict sharing